### PR TITLE
Temp disable all tests that are failing in AppVeyor

### PR DESCRIPTION
**Pull request by mjbvz**
We need to get CI back into a good state so we can have some confidence when merging in pull requests. This change just disables all tests that are failing in appveyor. Related to #689

The major source of problems is the REPL tests, which seem to be failing nondeterministicly. I think there is because of `SocketLock` and threading. I just disabled them all for now but will revise this later.

### CODE REVIEW

**Comment by mjbvz** [comment]
I've opened #718 with some ideas of how to deal with the skipped unit tests.

File: Nodejs/Tests/Core/ReplWindowTests.cs [hunk 1]
```diff
@@ -428,6 +446,7 @@ public class ReplWindowTests {
         }
 
         [Ignore]
+        [Ignore]
```
Comment (mousetraps) [inline Nodejs/Tests/Core/ReplWindowTests.cs]: Hrm, this is already being ignored.

File: Nodejs/Tests/Core/ReplWindowTests.cs [hunk 2]
```diff
@@ -517,6 +538,7 @@ public class ReplWindowTests {
...
```
Comment (mousetraps) [inline Nodejs/Tests/Core/ReplWindowTests.cs]: already being ignored

**Comment by mousetraps** [comment]
For the tests that are *only* failing in AppVeyor, and not locally with the same configuration, let's instead use the vstest TestCaseFilter option to ignore them because we might be able to fix some of these without modifying the code itself.

**Comment by mousetraps** [comment]
Also, have you changed anything about the AppVeyor run lately? It looks like it's failing after 2 min, and it should very much pass for this PR :smiley:

**Comment by mjbvz** [comment]
Ok, that sounds like a good plan. Will send out the updated review shortly.

I have not intentionally changed anything in AppVeyor but it did seem to start failing yesterday. Let's see what happens when I update this PR.

**Comment by mjbvz** [comment]
I've marked the unreliable/failing appveyor only tests with the category `AppVeyorIgnore`. I also ignored a few other tests that really were failing locally.

**Pull request merged**